make.ps1

```diff
@@ -62,15 +59,16 @@ function create-smoke-tests() {
     . ./envrc.ps1
     $env:PACT_VERSION="$env:USERNAME".replace(' ','')
     $env:UPDATE_PRESCRIPTIONS=$update
-    $env:SIGNING_PRIVATE_KEY_PATH="../../../sign/eps_int_test_private.key"
-    $env:SIGNING_CERT_PATH="../../../sign/eps_int_test_certificate.crt"
+    $env:PACT_USE_BROKER=$broker
```

Comment:
Does the equivalent need adding to the makefile too?

Reply:
Will raise a new pr